python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -125,6 +125,11 @@ async def point_azel(
                 getattr(self.rem, self.ptg_name).cmd_azElTarget,
                 slew_timeout=slew_timeout,
             )
+        except salobj.AckError as ack_err:
+            self.log.error(
+                f"Command to slew to azEl target rejected: {ack_err.ackcmd.result}"
+            )
+            raise ack_err
```

Comment:
This makes me a bit uneasy. It adds a lot of boilerplate if you will eventually want this for every command in this package. Usually the caller handles exceptions. Apparently that's not good enough here, but why? Perhaps RemoteCommand should log failed commands?

Reply:
So this is a complain I got from some users. The exception is too much to parse for people that are not used to tracebacks. I don't know about logging it from the RemoteCommand. Might be a good idea if the `ackcmd.result` contains enough information.

Reply:
Oy. OK. I'll think about logging in RemoteCommand.